crates/uv-cli/src/lib.rs

```diff
@@ -568,6 +568,14 @@ pub struct PipCompileArgs {
     #[arg(long)]
     pub python_platform: Option<TargetTriple>,
 
+    /// Perform a universal resolution, attempting to generate a single `requirements.txt` output
+    /// file that is compatible with all Python platforms.
```

Comment:
"Python platforms" is likely unclear to the users, we likely need something along the lines of "all operating systems, architectures and supported Python versions, as supported by your dependencies"